### PR TITLE
CMS5 compatibility

### PR DESCRIPTION
**Pull request by pack254**
Upgrade the module to be compatible with CMS 5

### CODE REVIEW

**Comment by thats4shaw** [comment]
@pack254 looks like there is some unit test issues we will need to work through.  Let me know if you want to have a crack first otherwise I'm happy to tidy them up.

**Comment by pack254** [comment]
> @pack254 looks like there is some unit test issues we will need to work through. Let me know if you want to have a crack first otherwise I'm happy to tidy them up.

@thats4shaw thank you for feedback, yep I will have a look into these issues and I will try to sort it out.

**Comment by pack254** [comment]
> > @pack254 looks like there is some unit test issues we will need to work through. Let me know if you want to have a crack first otherwise I'm happy to tidy them up.

> @thats4shaw thank you for feedback, yep I will have a look into these issues and I will try to sort it out.

File: _config/trailing-slash.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
```
Comment (thats4shaw) [inline _config/trailing-slash.yml]: @pack I see what you're trying to do here but we wouldn't want to enforce this config at a module level.  I'll push an alternative fix.

**Comment by thats4shaw** [comment]
@pack254 I've made a small tweak which is allowing the tests to pass.  If you could give it a spin on your project and then we will merge and tag!  Thank you for your work on this.

**Comment by pack254** [comment]
Thanks @thats4shaw for tweaking this up and it's good to know about the config 😃 . I test this branch with lincoln-ltl and it's working fine!. LGTM

**Pull request merged**